.env.example

```diff
@@ -2,4 +2,9 @@ TOKEN=
 CLIENT_ID=
 GUILD_ID=
 SOURCEJUMP_API_KEY=
-SOURCEJUMP_API_URL=
\ No newline at end of file
+SOURCEJUMP_API_URL=
+SQL_HOST=
+SQL_USER=
+SQL_DATABASE=
+SQL_PORT=
+SQL_PW=
```

Comment:
Can we change this to DATABASE_URL instead and use a MySQL connection string like `mysql://user:password@host/db`?